charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
+  - Engage with the [Compliance Working Group](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md) to ensure that the data release and data distribution processes are facilitating appropriate data access to all of the types of consumers
+  - Ensure released data are useable to tertiary portals and users can easily cross-reference across portals
+  - Determine best practices for when and how changes are made to DCP outputs after release
+
+- Community engagement
+  - Define data use policy and citation instructions for HCA data users
+  - Define mechanism(s) for which the community is notified when DCP products are updated after release
+  - Partner with Content and Portal Development to ensure clear, findable, and current documentation describing data release and data distribution
+  - Ensure effective communication of HCA data updates and user features to the scientific community
+  - Partner with UX to assess curation added value of data distribution through communities’ feedback, and better understand their expectations and needs
+  - Onboard staff to DCP ZenDesk ticketing system
+  - Take ownership of establishing process and interfacing with DCP components for assigning, responding, and integrating ZenDesk tickets
+  - Compile feedback gained from consumers through ZenDesk queries and comments, and report back to the DCP in order to facilitate system improvement 
+
+
+
+### Scientific "guardrails"
+
+In order to meet the needs of the communities of scientific consumers, the DataOps team requires input from the Governance Group as to where curated data distribution will add value for the researcher.
+
+## Out-of-scope
+
+The Data Operations is not responsible for determining what data or data types are made available through the HCA.
+
+As many of the features and requirements for data release and data distribution will be implemented through other components, the majority of technical aspects pertaining to data release and data distribution will be implemented by the appropriate component, and not Data Operations.
+
+## Milestones and Deliverables
+
+2019 Q3
+- Follow a dataset through the entire data workflow in order to understand each step and assess the current process. 
+- Organize incentivized data contribution program in conjunction with the HCA General Meeting in Barcelona in order to encourage pre-publication data contribution. 
+- Establish ZenDesk ticket triage procedure.
+
+2019 Q4
+- Write actional RFCs aimed to improve the data workflow.
```

Comment:
should actional be actionable?

Reply:
That's one interpretation. We think this was initially a typo but everyone looked it up and found out actional fit, too. But actionable is better - updated in ff41467